tests/test_tui_runtime_coverage.py

```
@@ -0,0 +1,196 @@
+def test_conpty_reader_does_not_retain_unbounded_post_startup_output() -> None:
+    """The startup marker must survive, while later PTY noise cannot grow memory."""
+
+    smoke_path = ROOT / "cmux-tui/scripts/smoke-windows-conpty-resize.py"
+    module_name = "cmux_tui_conpty_resize_smoke_for_contract_test"
+    winpty_stub = types.ModuleType("winpty")
+    winpty_stub.PtyProcess = object
+    previous_winpty = sys.modules.get("winpty")
+    sys.modules["winpty"] = winpty_stub
+    try:
+        spec = importlib.util.spec_from_file_location(module_name, smoke_path)
+        assert spec is not None and spec.loader is not None
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)